<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nFindPeaks</class>
 <widget class="QMainWindow" name="nFindPeaks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,1,0">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Scale</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="originOffset">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="neutrinoSave" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="setOrigin">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="scaleOffset">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="neutrinoSave" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="scale">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Origin</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="setScale">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="origin">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="direction">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="neutrinoSave" stdset="0">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>H</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>V</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QDoubleSpinBox" name="param">
         <property name="font">
          <font>
           <family>Lucida Sans</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
         <property name="neutrinoSave" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="nCustomPlotMouseX" name="plot" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadPref"/>
   <addaction name="actionSavePref"/>
   <addaction name="separator"/>
   <addaction name="actionSaveClipboard"/>
   <addaction name="actionSaveTxt"/>
   <addaction name="actionSavePDF"/>
   <addaction name="separator"/>
   <addaction name="actionRect"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <action name="actionLoadPref">
   <property name="icon">
    <iconset>
     <normaloff>:icons//loadPref.png</normaloff>:icons//loadPref.png</iconset>
   </property>
   <property name="text">
    <string>loadPref</string>
   </property>
   <property name="toolTip">
    <string>Load Preferences</string>
   </property>
  </action>
  <action name="actionSavePref">
   <property name="icon">
    <iconset>
     <normaloff>:icons//savePref.png</normaloff>:icons//savePref.png</iconset>
   </property>
   <property name="text">
    <string>savePref</string>
   </property>
   <property name="toolTip">
    <string>Save Preferences</string>
   </property>
  </action>
  <action name="actionSaveClipboard">
   <property name="icon">
    <iconset>
     <normaloff>:icons//saveClipboard.png</normaloff>:icons//saveClipboard.png</iconset>
   </property>
   <property name="text">
    <string>saveClipboard</string>
   </property>
   <property name="toolTip">
    <string>Copy to Clipboard</string>
   </property>
  </action>
  <action name="actionSaveTxt">
   <property name="icon">
    <iconset>
     <normaloff>:icons//saveTxt.png</normaloff>:icons//saveTxt.png</iconset>
   </property>
   <property name="text">
    <string>saveTxt</string>
   </property>
   <property name="toolTip">
    <string>Export TXT</string>
   </property>
  </action>
  <action name="actionSavePDF">
   <property name="icon">
    <iconset>
     <normaloff>:icons//savePDF.png</normaloff>:icons//savePDF.png</iconset>
   </property>
   <property name="text">
    <string>savePDF</string>
   </property>
   <property name="toolTip">
    <string>Export PDF or SVG</string>
   </property>
  </action>
  <action name="actionRect">
   <property name="icon">
    <iconset>
     <normaloff>:icons//rect.png</normaloff>:icons//rect.png</iconset>
   </property>
   <property name="text">
    <string>rect</string>
   </property>
   <property name="toolTip">
    <string>Show Region</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nCustomPlotMouseX</class>
   <extends>QWidget</extends>
   <header>nCustomPlots.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
